tests/drivers/clock_control/nrf_clock_control/src/main.c

```diff
@@ -9,13 +9,17 @@
 #include <zephyr/drivers/clock_control/nrf_clock_control.h>
 #include <zephyr/kernel.h>
 #include <zephyr/ztest.h>
+#if defined(CONFIG_CLOCK_CONTROL_NRF2_AUXPLL)
+#include <zephyr/dt-bindings/clock/nrf-auxpll.h>
+#endif
 
 struct test_clk_context {
 	const struct device *clk_dev;
 	const struct nrf_clock_spec *clk_specs;
 	size_t clk_specs_size;
 };
 
+#if defined(CONFIG_CLOCK_CONTROL_NRF2)
```

Comment:
This test only builds for nrf54h20 ATM, so this if guard should not be needed AFAIK